networkx/algorithms/components/strongly_connected.py

```diff
@@ -141,13 +140,12 @@ def kosaraju_strongly_connected_components(G, source=None):
 
     Examples
     --------
-    Generate a list of strongly connected components, in ascending order of
-    length.
+    Generate a list of strongly connected components of a graph:
```

Comment:
Here the previous docs say it yields in ascending order of length. The PR removes that phrase.
Was it incorrect?